Assets/Scripts/Gameplay/GameplayObjects/ClientTossAttackDisplay.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Unity.Netcode;
+using UnityEngine;
+
+namespace Unity.Multiplayer.Samples.BossRoom.Client
+{
+    public class ClientTossAttackDisplay : NetworkBehaviour
```

Comment:
From the name only, not sure I understand what this class does and what it's for. what is it and why does it follow the tossed item?

Reply:
This is for the toss attack radius world-space display that is shown on the floor, similar to the AoE action.

We certainly can decouple these two. Though, some of the NetworkObjects we're already spawning have this hierarchy (arrows): root is network/physics object, and child is graphics object.

My opinion is that this should stay as part of the tossed item prefab as a child graphics object, and any explosion effect can be instantiated locally (callback to that state vs rpc discussion).

Reply:
My comment was mostly about the naming and clarity. I think the structure is fine and should stay consistent with other boss room things. The way it's named just sounds like it's an action, when it's actually _something_ --> the bomb danger circle.